tests/Makefile

```diff
@@ -5,8 +5,8 @@ RRTMGP_BUILD = $(RRTMGP_ROOT)/build
 #
 # RRTMGP library, module files
 #
-LDFLAGS   += -L$(RRTMGP_BUILD)
-LIBS      += -lrrtmgp -lrte
+# LDFLAGS   += -L$(RRTMGP_BUILD)
```

Comment:
Here too I'm confused as to why the Makefiles are being modified

Reply:
Its the duplicate static linking problem that the GPU linker cannot handle. See previous comment.